requirements.txt

```diff
@@ -29,3 +29,5 @@ graphene==2.1.8
 graphene-django==2.15.0
 
 pytz==2021.1
+
+shortuuid>=1.0.1
```

Comment:
Can we add a short comment after this to say why its imported (just helps avoid clutter when we check if we can get rid of any)